src/dialogs/more-info/controls/more-info-light.ts

```diff
@@ -573,52 +235,33 @@ class MoreInfoLight extends LitElement {
         align-items: center;
       }
 
-      .content > * {
-        width: 100%;
-      }
-
-      .color_temp {
-        --ha-slider-background: -webkit-linear-gradient(
-          var(--float-end),
-          rgb(166, 209, 255) 0%,
-          white 50%,
-          rgb(255, 160, 0) 100%
-        );
-        /* The color temp minimum value shouldn't be rendered differently. It's not "off". */
-        --paper-slider-knob-start-border-color: var(--primary-color);
-        margin-bottom: 4px;
-      }
-
-      .segmentationContainer {
-        position: relative;
-        max-height: 500px;
+      .buttons {
         display: flex;
+        align-items: center;
         justify-content: center;
+        margin-bottom: 12px;
       }
-
-      ha-button-toggle-group {
-        margin-bottom: 8px;
+      .buttons > * {
+        margin: 4px;
       }
 
-      ha-color-picker {
-        --ha-color-picker-wheel-borderwidth: 5;
-        --ha-color-picker-wheel-bordercolor: white;
-        --ha-color-picker-wheel-shadow: none;
-        --ha-color-picker-marker-borderwidth: 2;
-        --ha-color-picker-marker-bordercolor: white;
+      ha-more-info-light-brightness,
+      ha-more-info-light-toggle {
+        margin-bottom: 24px;
       }
 
-      .segmentationButton {
-        position: absolute;
-        top: 5%;
-        left: 0;
-        color: var(--secondary-text-color);
+      ha-attributes {
+        width: 100%;
       }
 
-      hr {
-        border-color: var(--divider-color);
-        border-bottom: none;
-        margin: 16px 0;
+      md-outlined-icon-button-toggle,
+      md-outlined-icon-button {
+        --md-sys-color-on-surface: var(--secondary-text-color);
+        --md-sys-color-on-surface-variant: var(--secondary-text-color);
+        --md-sys-color-on-surface-rgb: var(--rgb-secondary-text-color);
+        --md-sys-color-outline: var(--secondary-text-color);
+        --md-sys-color-inverse-surface: var(--primary-text-color);
+        --md-sys-color-inverse-on-surface: var(--primary-background-color);
```

Comment:
Should we set these globally?

Reply:
We should add this globally when we will have a `ha-outlined-button`.